.gitignore

```diff
@@ -93,3 +93,6 @@ staticfiles/
 release-notes-checklist
 
 static/scss/material.css
+
+# Local settings
+micromasters.yml
```

Comment:
What's this for? It's fine to leave it, just wondering if this is forthcoming in the future